instance creation
on: anError
	^ self new error: anError